chart/templates/drupal-configmap.yaml

```diff
@@ -237,6 +237,8 @@ data:
         add_header                  Strict-Transport-Security max-age=31536000;
         add_header                  X-XSS-Protection '1; mode=block';
 
+        map_hash_bucket_size 128;
```

Comment:
Why is this needed?

Reply:
because nginx didn't start and returned this error: 
> Nginx: could not build map_hash, you should increase map_hash_bucket_size: 64

https://serverfault.com/questions/853102/nginx-could-not-build-map-hash-you-should-increase-map-hash-bucket-size-64
So i set it to the next value or 128 and that helped. 
http://nginx.org/en/docs/http/ngx_http_map_module.html#map_hash_bucket_size

Reply:
If anyone stumbles accross an error, we can extract the `map_hash_bucket_size` as a variable, but until then, I vote for keeping it simple.